metal/spi.h

```diff
@@ -21,6 +21,14 @@ struct metal_spi_config {
     unsigned int cs_active_high : 1;
     /*! @brief The chip select ID to activate for the SPI transfer */
     unsigned int csid;
+    /*! @brief The spi command frame number (cycles = num * frame_len) */
+    unsigned int cmd_num;
+    /*! @brief The spi address frame number */
+    unsigned int addr_num;
+    /*! @brief The spi dummy frame number */
+    unsigned int dummy_num;
+    /*! @brief The Dual/Quad spi mode selection.*/
+    enum { QSPI_ALL, QSPI_DATA_ONLY, QSPI_ADDR_DATA } qspi_mode;
```

Comment:
If this applies to both dual and quad-SPI, we should probably not name it "qspi_mode".

Reply:
Any suggestion for this variable name? How about "multi_wire" or "multi_wire_mode"?